packages/database/src/modelClasses/GeographicalArea.js

```diff
@@ -23,10 +23,6 @@ class GeographicalAreaType extends DatabaseType {
     return this.model.findById(this.parent_id);
   }
 
-  async children() {
-    return this.model.find({ parentId: this.id });
```

Comment:
When fixing the `parentId` -> `parent_id` in `Entity.js`, I searched to find if that mistake existed elsewhere - which it did here (historical), but the `children()` function doesn't appear to be used, so I've removed it entirely.